app/scripts/modules/core/src/cluster/AllClustersGroupings.tsx

```diff
@@ -116,10 +128,12 @@ export class AllClustersGroupings extends React.Component<IAllClustersGroupingsP
             height={height}
             width={width}
             rowCount={groups.length}
-            // deferredMeasurementCache={this.cellCache}
+            deferredMeasurementCache={this.cellCache}
```

Comment:
If you are curious what this line does, so am I

Reply:
huh

Reply:
this just makes it to where the `<List>` is aware of its children's cell measurements, which let's it do a better job of estimating the overall height of the list, etc.